src/components/overlayCookie/overlayCookie.module.scss

```diff
@@ -4,31 +4,12 @@
 
 
 .cookieWrapper {
-  width: 100%;
-  position: fixed;
-  bottom: 0;
   background-color: $color-tertiary;
```

Comment:
Please use the tailwind colors

Reply:
Yes, I'll check now how things are done! :+1: @crysfel